packages/react/src/primitives/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
+
+import {
+  BaseBreadcrumbProps,
+  BreadcrumbsProps,
+  Primitive,
+  ForwardRefPrimitive,
+} from '../types';
+
+import { BreadcrumbItem } from './BreadcrumbItem';
+import { BreadcrumbLink } from './BreadcrumbLink';
+import { BreadcrumbSeparator } from './BreadcrumbSeparator';
+import { BreadcrumbContainer } from './BreadcrumbContainer';
+
+const BreadcrumbsPrimitive: Primitive<BreadcrumbsProps, 'nav'> = (
+  { className, items, separator = BreadcrumbSeparator, ...rest },
+  ref
+) => {
+  const ariaLabel = rest['aria-label'] ?? 'Breadcrumb';
+
+  return (
+    <BreadcrumbContainer aria-label={ariaLabel} className={className} ref={ref}>
```

Comment:
Might be intentional and if so feel free to disregard but should `rest` be passed to the `BreadcrumbsContainer`?
```suggestion
    <BreadcrumbContainer
      {...rest}
      aria-label={ariaLabel}
      className={className}
      ref={ref}
    >
```